### ISSUE TITLE
PDF Files "could not open x.pdf" supported file has been damaged or wasn't correctly decoded

### ISSUE THREAD

**Issue by sma92878** [trustedsec/social-engineer-toolkit#369]
![could not open](https://cloud.githubusercontent.com/assets/22758330/23219652/acbe6240-f8ed-11e6-8191-48305109a698.PNG)

I have installed SET Version 7.5.1 with Metasploit version 4.13.20

Every PDF I have generated thus far gives the above error.  I have tried various payloads CoolType SING - was the main one I used based on a PACKT book I'm going through.  I've even copied the PDFs directly from the source machine to my Windows target without email.  I've tried and Acrobar view 10 and the latest version DC, all with the same result.  

The target machine is Windows 7, creating machine the latest version of Ubuntu.

Thank you kindly
/hugs

**Comment by sma92878** [comment]
Update:  I have continued to try and get this to work, when I try and open the PDF document on my Ubuntu system the document shows up as a black PDF.  This included when I select a template PDF to use of my own.

Thanks again

**Comment by sma92878** [comment]
Continuing to try and resolve this on my own, I have a further update.  I did another sample based on a PDF I generated as opposed to one I found online.  I've noticed that when I select the:

Adobe CoolType SING Table "uniqueName" overflow the PDF document simply says "Hello World"

I don't believe I'm typing the path wrong, as I've tested this.  I've noticed there's an error notice and SET tells you it's falling back to the default blank PDF.

![hello world](https://cloud.githubusercontent.com/assets/22758330/23265353/579ab32a-f9b2-11e6-96bb-6fbbcf172c4e.PNG)

Thank you kindly
